vm-sysroot/image/build-image.sh

```diff
@@ -6,39 +6,16 @@ if [ "$$" != 1 ] && [ "x$1" != x ]
 then cat >&2 << EOF
 usage: image/build-image.sh
 
-Build the disk image image/hdd.img, using sysroot in ./sysroot. Uses UML to isolate itself.
+Build the disk image image/hdd.img, using sysroot in ./sysroot.
```

Comment:
Is this sysroot (and its layout) documented?

Reply:
This sysroot is just the resulting image's root filesystem. It is build using `./build.sh` or one of its wrappers.